src/utils/update/company.profile.mapper.to.oversea.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { CompanyProfile, OfficeAddress } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { yesNoResponse } from "../../model/data.types.model";
+import { Entity } from "../../model/entity.model";
+
+export const mapCompanyProfileToOverseasEntity = (cp: CompanyProfile): Entity => {
+  return {
+    name: cp.companyName,
```

Comment:
The `name` field has been removed from `entity` object recently, you should have a type error for that.

Reply:
I can see it has been removed. Main has been merged and fixed